src/Tizen.Multimedia.Vision/MediaVision/SurveillanceEngine.cs

```diff
@@ -35,6 +35,7 @@ public abstract class SurveillanceEngine : IDisposable
         private IntPtr _handle = IntPtr.Zero;
         private bool _disposed = false;
 
+        internal EventOccurredCallback OnEventDetected;
```

Comment:
It seems that Recognized and Detected events in several classes are using this sindle field together. Is it OK?

Reply:
I missed that all those classes are not just seperate instance but base and derived class relation.  It's OK.